airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -105,6 +112,160 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
 
   const columnHelper = createColumnHelper<FullTableProps>();
 
+  /**
+   * TODO:
+   * [x] only calculate this if the flag is on!
+   * [x] link to connection from connection name
+   * [x] sizing of columns
+   * [x] alignment of headers/cells
+   * [x] sorting
+   * [x] horizontal overflow
+   * [ ] cells should truncate + show a tooltip? (long connection/connector names are a problem currently)
+   * [ ] convert link to new Link component once it's merged
+   * [ ] if we want the source and destination names to link to their pages, we'll need the
+   *     source/destination id's (not their definition id's)
+   * [ ] page header in designs has a link; confirm target and add!
+   * [ ] integrate free connector program data
+   * [ ] daily breakdown per connection
+   * */
+
+  const billingInsightsColumns = React.useMemo(() => {
+    return !isBillingInsightsEnabled
+      ? null
+      : [
+          columnHelper.accessor("connectionName", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("connectionName")}
+                isActive={sortBy === "connectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.connection" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
```

Comment:
This (and the source + destination cells) should have a Tooltip if they have an ellipsis.

We have a pattern [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-webapp/src/components/connection/CatalogTree/next/CatalogTreeTableCell.tsx), which makes me wonder if this is something we want to extract for more generic use.  I am leaning yes?